Wikipedia/UI-V5/WMFMinimalArticleContentCell.m

```diff
@@ -2,65 +2,25 @@
 //  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
 
 #import "WMFMinimalArticleContentCell.h"
-#import "WMFLinkButtonFactory.h"
-
-#import <Masonry/Masonry.h>
-#import <DTFoundation/DTTiledLayerWithoutFade.h>
-
-@interface DTAttributedTextContentView (WMFOverrideLayerClass)
-
-@end
-
-@implementation DTAttributedTextContentView (WMFOverrideLayerClass)
-
-+ (void)load {
-    /*
-       Set tiled layers for all attributed text content views. This prevents rendering all attributed text at once, but
-       might require unnecessary overrhead for smaller layers.
-
-       If there's a situation where the amount of text to shown is likely to be small most of the time, consider using
-       a custom subclass of DTAttributedTextContentView which returns `CALayer` from `layerClass` w/o checking super.
-     */
-    [DTAttributedTextContentView setLayerClass:[DTTiledLayerWithoutFade class]];
-}
-
-@end
-
 
 @interface WMFMinimalArticleContentCell ()
-<DTAttributedTextContentViewDelegate>
 
-@property (nonatomic, strong) WMFLinkButtonFactory* buttonFactory;
+@property (nonatomic, strong) IBOutlet UITextView* textView;
```

Comment:
not sure if this is a "best practice" or not, but I usually make subviews `weak` since they're retained by their superview